src/OrleansServiceBus/Providers/Streams/EventHub/IEventHubSettings.cs

```diff
@@ -8,6 +8,13 @@ public interface IEventHubSettings
         string ConsumerGroup { get; }
         string Path { get; }
         int? PrefetchCount { get; }
+
+        /// <summary>
+        /// Indicates if stream provider should read all new data in partition, or from start of partition.
+        /// True - read all new data added to partition.
+        /// False - start reading from beginning of partition.
+        /// Note: If checkpoints are used, stream provider will always begin reading from most recent checkpoint.
+        /// </summary>
         bool StartFromNow { get; }
```

Comment:
thanks for the comments. Nevertheless, this highlights whether it's appropriate to use a bool flag instead of a more explicit enum. No need address now though as it's not a big deal, just something I forgot to comment on when this was initially added, and this addition reminded me.